pkg/codegen/docs/gen.go

```diff
@@ -1036,19 +1066,22 @@ func (mod *modContext) genResource(r *schema.Resource) resourceDocArgs {
 	// Create a resource module file into which all of this resource's types will go.
 	name := resourceName(r)
 
-	// TODO: Unlike the other languages, Python does not have a separate Args object for inputs.
-	// The args are all just named parameters of the constructor. Consider injecting
-	// `resource_name` and `opts` as the first two items in the table of properties.
 	inputProps := make(map[string][]property)
 	outputProps := make(map[string][]property)
 	stateInputs := make(map[string][]property)
+
+	var filteredOutputProps []*schema.Property
```

Comment:
These changes don't have any effect on the generated resource docs. I just happened to notice that we don't need to calculate the filtered properties per-language.